ZividOpenCV/CaptureUndistortRGB/CaptureUndistortRGB.cpp

```diff
@@ -25,60 +29,46 @@ int main()
         std::cout << "Connecting to the camera" << std::endl;
         auto camera = zivid.connectCamera();
 
-        std::cout << "Configuring the camera settings" << std::endl;
-        camera << Zivid::Settings::Iris{ 21 } << Zivid::Settings::ExposureTime{ std::chrono::microseconds{ 20000 } }
-               << Zivid::Settings::Gain{ 1 } << Zivid::Settings::Brightness{ 1.0 };
-
-        std::cout << "Capturing a frame" << std::endl;
-        const auto frame = camera.capture();
-
-        std::cout << "Setting up visualization" << std::endl;
-        Zivid::CloudVisualizer vis;
-        zivid.setDefaultComputeDevice(vis.computeDevice());
-
-        std::cout << "Displaying the point cloud" << std::endl;
-        vis.showMaximized();
-        vis.show(frame);
-        vis.resetToFit();
-
-        std::cout << "Running the visualizer. Blocking until the window closes" << std::endl;
-        vis.run();
-
-        std::cout << "Converting ZDF point cloud to OpenCV format" << std::endl;
+        std::cout << "Enter <2d><Return> for 2D only, and just <Return> for 3D" << std::endl;
```

Comment:
<2d><Return> is not the most intuitive description. I had to stop and remember that <Return> means hitting the 'Enter' key. It seems that it is traditionally 'return' on Mac PC and 'enter' on Windows.

It would've been more intuitive to me (this doesn't mean it would've be the best way) if it was like this: "Enter "2d" and press enter/return key for 2D only..."

Is the way you are describing options with "<>" a common approach? If it is, leave it (and educate me). If not, try to make it simpler (idiot-proof).

Reply:
I have no objection to what you propose. No reason not to be more explicit.